calico-cloud/operations/license-options.mdx

```diff
@@ -12,7 +12,7 @@ Review options for tracking {{prodname}} license expiration.
 
 ## Concepts
 
-We highly recommend using the [license agent using Prometheus](monitor/metrics/license-agent.mdx) to get alerts on your {{prodname}} license expiration day to avoid disruption to services. Regardless if you using the alerting feature, here are some things you should know.
+We highly recommend using the [license agent using Prometheus](monitor/metrics/license-agent.mdx) to get alerts on your {{prodname}} license expiration day to avoid disruption to services. regardless of whether you using the alerting feature, here are some things you should know.
```

Comment:
```suggestion
We highly recommend using the [license agent using Prometheus](monitor/metrics/license-agent.mdx) to get alerts on your {{prodname}} license expiration day to avoid disruption to services. Regardless of whether you using the alerting feature, here are some things you should know.
```

Reply:
see other instances below

Reply:
Oops. I even knew about that when I first saw it,  I'll fix tonight.